Sources/Hypertext.swift

```diff
@@ -6,18 +6,23 @@
 //  Copyright © 2016 Sahand Nayebaziz. All rights reserved.
 //
 
-public protocol Renderable {
-    func render() -> String
-    func render(startingWithSpacesCount: Int) -> String
```

Comment:
This PR does offer that but by way of an extension to `Renderable` that takes an optional argument

```
public protocol Renderable: CustomStringConvertible {
    func render(indent: Int?, startingWithSpacesCount: Int) -> String
}

public extension Renderable {
    ...

    func render(indent: Int? = nil) -> String {
        return render(indent: indent, startingWithSpacesCount: 0)
    }
}
```

Sorry for being confusing, I just wanted to explain my thinking for you first before asking for something :)

I am asking for us to keep `Renderable` as:

```swift
public protocol Renderable {
  func render() -> String
  func render(startingWithSpacesCount: Int) -> String // But let's work on this one and change it to take the space/indent like you have
}
```